test/lib/processRequest.test.js

```diff
@@ -55,6 +55,59 @@ module.exports = tests => {
     }
   })
 
+  tests.add(
+    '`processRequest` with a single file and `createReadStream` options.',
+    async () => {
+      let serverError
+
+      const server = http.createServer(async (request, response) => {
+        try {
+          const operation = await processRequest(request, response)
+
+          ok(operation.variables.file instanceof Upload)
+
+          const upload = await operation.variables.file.promise
+
+          strictEqual(upload.filename, 'a.txt')
+          strictEqual(upload.mimetype, 'text/plain')
+          strictEqual(upload.encoding, '7bit')
+
+          const stream = upload.createReadStream({
+            encoding: 'base64',
+            highWaterMark: 100
+          })
+
+          ok(stream instanceof ReadStream)
+          ok(stream.readableHighWaterMark, 100)
```

Comment:
This is incorrect use of the [`ok`](https://nodejs.org/api/assert.html#assert_assert_ok_value_message) assertion, which only tests the first parameter is truthy.